README.rst

```diff
@@ -8,48 +8,52 @@ Please tag **@edx/arch-team** on any PRs or issues.  Thanks.
 Introduction
 ------------
 
-Micro-frontend for the single-page payment/checkout process. This application supports Apple Pay, PayPal, and Cybersource credit card payment types.
+Micro-frontend for the single-page payment/checkout process; order history and receipt pages are still served by the ecommerce service. This application only supports PayPal and Cybersource credit card payments.
 
 Getting Started
 ---------------
 
-**Prerequisite**
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
 
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
+Once set up, this frontend and all its prerequisites can be started with:
 
-**Installation and Startup**
+.. code-block::
 
-1. Clone this repo:
+  cd devstack
+  make dev.up.frontend-app-payment
 
-  ``git clone https://github.com/edx/frontend-app-payment.git``
+The frontend runs at `http://localhost:1998 <http://localhost:1998>`_, though it is rare to interact with it directly.  Rather, log into the LMS at http://localhost:18000/login and enroll in a course, choosing to upgrade to be taken to the checkout page.
 
-2. Install npm dependencies:
-
-  ``cd frontend-app-payment && npm ci``
+Development
+-----------
 
-  Note: ``npm ci`` is recommended over ``npm install`` to match the way CI and production builds work and avoid unintentional changes to ``package_lock.json`` when doing other work.
+To work on this frontend: with the prerequisites running as above, bring down the ``frontend-app-payment`` container, then start the development server *outside* Docker as follows:
```

Comment:
Would it be worth explaining why if by default, this app is started up automatically with devstack, but if you do development work, you need to start it up outside of devstack?

Reply:
I don't really understand why that is the case myself; I'm going to leave this as-is and get it merged.

Reply:
The only reason I'm aware of is if you need to apply custom package overrides locally via `module.config.js` (e.g. you need the edx-brand theme or you're working on a new Paragon component and need to see how it interacts on an MFE page).